### PR TITLE
Update Jenkins-Proxy

### PR DESCRIPTION
**Pull request by sthaha**
Updates include:
  - CORS enabled for new api endpoint
  - Improved logging (adds request-id)
  - Reverse-Proxy responds before the gateway times out (solves 504 issue)

### CODE REVIEW

**Comment by jmelis** [comment]
[test]

File: dsaas-services/f8-jenkins-idler.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
   - name: staging
     parameters:
       IMAGE: prod.registry.devshift.net/osio-prod/fabric8-services/fabric8-jenkins-idler
-- hash: a6bcdf660e96c0599e7aea9e20037afe3eb9cc2c
+- hash: 6e9fcefc0451bccbfeaa07b825d309b9dc9eb495
```
Comment (jmelis) [inline dsaas-services/f8-jenkins-idler.yaml]: shouldn't this go in the jenkins-proxy section?

Comment (sthaha) [reply dsaas-services/f8-jenkins-idler.yaml]: yes, it must be proxy did I update the wrong section??

Comment (sthaha) [reply dsaas-services/f8-jenkins-idler.yaml]: @jmelis it seems I did update the right section.

Comment (jmelis) [reply dsaas-services/f8-jenkins-idler.yaml]: You are very right! I'm sorry

**Pull request merged**